client/src/components/util/version.ts

```diff
@@ -0,0 +1,11 @@
+export function checkValidVersion(corporaProps: any) {
+  const isValidVersionOne =
+    ["1.0.0", "1.1.0"].indexOf(corporaProps?.version?.corpora_schema_version) >
```

Comment:
super nit:

Hoist constants out of the function declaration, so we don't need to create new arrays every time:

```ts
const VERSION_ONES = ["1.0.0", "1.1.0"];
const VERSION_TWOS = ["2.0.0"];

export function checkValidVersion(corporaProps: any) {
  if (!corporaProps) return false;

  const { version, schema_version } =  corporaProps;

  const isValidVersionOne = VERSION_ONES.includes(version?.corpora_schema_version);
  const isValidVersionTwo = VERSION_TWOS.includes(schema_version);

  return isValidVersionOne || isValidVersionTwo; 
}
```

Reply:
Good point, thanks!